core/src/main/java/org/polypheny/db/adapter/annotations/AdapterProperties.java

```diff
@@ -29,6 +29,8 @@
 
     String name();
 
+    String logo();
+
```

Comment:
Could you rename this to be a bit more specific what kind this actually is, something like `logoPath`, `logoResource`,`logoAssetPath`, etc. Ideally something with points to the fact, that it either is a resource or a class text.

Reply:
Ohh I see, thank you so much for your suggestion! I will update it to `logoAssetPath`